Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:18-alpine
+LABEL org.opencontainers.image.source https://github.com/filecoin-station/core
+USER node
+WORKDIR /usr/src/app
```

Comment:
I believe it will be `~/.local/state/`, as that's the default `$XDG_STATE_HOME`. Do you think it should be somewhere else? Because if it should be somewhere else, we might consider also storing it somewhere else outside of Docker

Reply:
> I believe it will be `~/.local/state/`, as that's the default `$XDG_STATE_HOME`.

Do we know the home dir (`~`) for our `node` user? 

Why I am asking: to inspect logs and other state files from outside, we need to know where to find those files inside the docker container.

Maybe we don't need that because users can run `station metrics` and `station logs` instead?

> Do you think it should be somewhere else? Because if it should be somewhere else, we might consider also storing it somewhere else outside of Docker

I was considering this too, but I don't know if storing the state outside of Docker is necessary.

I think the current setup is good enough and further improvements can wait until we get feedback from people running Station Core via Docker.

Reply:
> Maybe we don't need that because users can run station metrics and station logs instead?

That's my thinking too, the user doesn't need to know